console/src/components/Main/Actions.js

```diff
@@ -41,6 +47,63 @@ const loadMigrationStatus = () => dispatch => {
   );
 };
 
+const loadServerVersion = () => dispatch => {
+  const url = Endpoints.version;
+  const options = {
+    method: 'GET',
+    credentials: globalCookiePolicy,
+    headers: { 'Content-Type': 'application/json' },
+  };
+  return dispatch(requestActionPlain(url, options)).then(
+    data => {
+      let parsedVersion;
+      try {
+        parsedVersion = JSON.parse(data);
+      } catch (e) {
+        console.error(e);
+      }
+      dispatch({
+        type: SET_SERVER_VERSION_SUCCESS,
+        data: parsedVersion.version,
+      });
+    },
+    error => {
+      console.error(error);
```

Comment:
Do we want to show to the user that there is `something wrong` with fetching the current API version

`please get back to us if this keeps happening. This can happen if the graphql engine api is down or is returning a 400.` might help